opentelemetry-api/src/global/metrics.rs

```diff
@@ -26,9 +29,9 @@ impl fmt::Debug for GlobalMeterProvider {
 impl MeterProvider for GlobalMeterProvider {
     fn versioned_meter(
         &self,
-        name: &'static str,
-        version: Option<&'static str>,
-        schema_url: Option<&'static str>,
+        name: Cow<'static, str>,
```

Comment:
What was the reason for not using `impl Into<Cow<'static, str>>` again?

Reply:
I can use impl Into here, but not for the Option types unfortunately, as you cannot convert &str into `Option<Cow<...>>`.

So the best i can do is just the first parameter for name. Would you like me to make this change?